sui/src/config/mod.rs

```diff
@@ -431,15 +439,12 @@ pub fn make_default_narwhal_committee(
 ) -> Result<ConsensusCommittee<Ed25519PublicKey>, anyhow::Error> {
     let mut ports = Vec::new();
     for _ in authorities {
-        let mut authority_ports = Vec::new();
```

Comment:
Say can you add a better RustDoc to this method, that it should only be for testing/

Reply:
I don't know the intent of this function, I'll let @asonnino comment on it's intent since it's narwhal related